<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan BÃ¶hmann</author>
 <class>AccountSettingsDialog</class>
 <widget class="QDialog" name="AccountSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string>icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Account Settings</string>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="buddy">
            <cstring>name</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="name"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Institution:</string>
           </property>
           <property name="buddy">
            <cstring>institution</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Holder name:</string>
           </property>
           <property name="buddy">
            <cstring>owner</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KLineEdit" name="owner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>5</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Account number:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>accountNumber</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KLineEdit" name="accountNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>5</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IBAN:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>iban</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="KLineEdit" name="iban">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>5</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string>The International Bank Account Number (IBAN) is an international standard for identifying bank accounts across national borders.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Opening Date:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>openingDate</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDateEdit" name="openingDate">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>134</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Opening Balance:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>openingBalance</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="openingBalance">
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.989999994635582</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>129</width>
               <height>26</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="institution"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>BIC:</string>
           </property>
           <property name="buddy">
            <cstring>bic</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KLineEdit" name="bic"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="accountLimitGroupBox">
         <property name="title">
          <string>Account Limits</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="useInLimit">
            <property name="text">
             <string>Maximum balance:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="inLimit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <double>99999999.989999994635582</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="useOutLimit">
            <property name="text">
             <string>Minimum balance:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="outLimit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>99999999.989999994635582</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="categories_tab">
      <attribute name="title">
       <string>Categories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeView" name="categoriesTreeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="addCategoryButton">
           <property name="toolTip">
            <string>Add new category</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addSubCategoryButton">
           <property name="toolTip">
            <string>Add new sub-category</string>
           </property>
           <property name="text">
            <string>Add child</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="importCategoryButton">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportCategoryButton">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="removeCategoryButton">
           <property name="toolTip">
            <string>Delete selected category</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name</tabstop>
  <tabstop>institution</tabstop>
  <tabstop>bic</tabstop>
  <tabstop>owner</tabstop>
  <tabstop>accountNumber</tabstop>
  <tabstop>iban</tabstop>
  <tabstop>openingDate</tabstop>
  <tabstop>openingBalance</tabstop>
  <tabstop>useInLimit</tabstop>
  <tabstop>inLimit</tabstop>
  <tabstop>useOutLimit</tabstop>
  <tabstop>outLimit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useInLimit</sender>
   <signal>clicked(bool)</signal>
   <receiver>inLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useOutLimit</sender>
   <signal>clicked(bool)</signal>
   <receiver>outLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
